WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -59,6 +63,16 @@ extension WordPressAuthenticationManager {
         // Ref https://github.com/wordpress-mobile/WordPress-iOS/pull/12332#issuecomment-521994963
         let enableSignInWithApple = !(BuildConfiguration.current ~= [.a8cBranchTest, .a8cPrereleaseTesting])
 
+        let googleLogingWithoutSDK: Bool = {
+            switch BuildConfiguration.current {
+            case .appStore:
+                // Rely on the remote flag in production
+                return RemoteFeatureFlag.sdkLessGoogleSignIn.enabled(using: remoteFeaturesStore)
+            case _:
```

Comment:
I guess it makes sense that the `_` syntax would compile here. But I think `default` is more suitable for "all other cases"?

Reply:
_But look at how homogeneous it looks with only `case`._ 😄 

I never dug this up, but I don't see any mention of `case _` in the [Swift book](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/enumerations/). Also taking into account the more relatively recent [`@unknown default` syntax](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/statements#Switching-Over-Future-Enumeration-Cases), it makes sense to prefer `default`.

Reply:
I believe the usage of `_` here falls into [the Wildcard Pattern](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/patterns/#Wildcard-Pattern)